### PR TITLE
Add rchavarria as maintainer

### PR DESCRIPTION
**Pull request by rchavarria**
I'm not very proud of adding myself as a maintainer, but somebody needs to do the job 😉 

Come on @exercism/ecmascripts, let's prepare for Nextercism.

### CODE REVIEW

**Comment by joelwallis** [comment]
I'll add myself as maintainer too 🙂

File: config/maintainers.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
     {
       "github_username": "rchavarria",
-      "show_on_website": false,
-      "alumnus": false,
-      "name": null,
-      "bio": null,
-      "link_text": null,
-      "link_url": null,
+      "show_on_website": true,
+      "alumnus": true,
```
Comment (joelwallis) [inline config/maintainers.json]: Hey @rchavarria,

@matthewmorgan is right. `alumnus` is meant to identify a maintainer that has been away for some time - and that's not really your case 😅 so what about making it `false` again?

Comment (rchavarria) [reply config/maintainers.json]: Whoops 😅

I'll update it.

**Comment by rchavarria** [comment]
@joelwallis what about it now?

**Comment by joelwallis** [comment]
Looks good!

**Pull request merged**